src/BIMDataComponents/BIMDataTable/BIMDataTable.vue

```diff
@@ -26,7 +26,15 @@
                 textAlign: column.align || 'left',
               }"
             >
-              {{ column.id ? column.label || column.id : column }}
+              <div :style="{ display: 'inline-flex' }">
+                {{ column.id ? column.label || column.id : column }}
+                <slot name="header-right"></slot>
+                <HeaderTableFilter
```

Comment:
What about `ColumnFilters` instead of `HeaderTableFilter` ?

Reply:
I propose `ColumnSorting` instead of `ColumnFilters`, as we are going to add another filter feature

Reply:
OK, sound good to me.